gammapy/spectrum/utils.py

```diff
@@ -294,7 +297,8 @@ def _trapz_loglog(y, x, axis=-1, intervals=False):
         # powerlaw integration
         trapzs = np.where(
             np.abs(b + 1.) > 1e-10, (y[slice1] * (
-                x[slice2] * (x[slice2] / x[slice1]) ** b - x[slice1])) / (b + 1),
+                x[slice2] * (x[slice2] / x[slice1]) ** b - x[slice1])) / (
```

Comment:
Can you undo that line break, even if the line is >80 characters, the first version was better to read...